index.d.ts

```diff
@@ -0,0 +1,37 @@
+interface INothing {
+  (): INothing;
+  (p1: any): INothing;
+  (p1: any, p2: any): INothing;
+  (p1: any, p2: any, p3: any): INothing;
+  (p1: any, p2: any, p3: any, p4: any): INothing;
+  (p1: any, p2: any, p3: any, p4: any, p5: any): INothing;
+  (p1: any, p2: any, p3: any, p4: any, p5: any, p6: any): INothing;
+  (p1: any, p2: any, p3: any, p4: any, p5: any, p6: any, p7: any): INothing;
+  (p1: any, p2: any, p3: any, p4: any, p5: any, p6: any, p7: any, p8: any): INothing;
+  (p1: any, p2: any, p3: any, p4: any, p5: any, p6: any, p7: any, p8: any, p9: any): INothing;
+  (p1: any, p2: any, p3: any, p4: any, p5: any, p6: any, p7: any, p8: any, p9: any, p10: any): INothing;
+  [key: string]: INothing;
+}
+
+interface IFunctionProps {
+  length: 0;
+  name: string;
+  prototype: object;
+  toLocaleString: "";
+  toString: "";
+  valueOf(): false;
```

Comment:
Should it have brackets?

Reply:
Yep, I forget parentheses on `toString` and `toLocaleString`
Already added
Thanks for notice, good catch!